apps/api/src/app/events/e2e/echo-trigger.e2e-ee.ts

```diff
@@ -320,7 +320,7 @@ async function discoverAndSyncEcho(session: UserSession) {
   const resultDiscover = await axios.get(echoServer.serverPath + '/echo?action=discover');
 
   await session.testAgent.post(`/v1/echo/sync`).send({
-    chimeraUrl: echoServer.serverPath + '/echo',
+    bridgeUrl: echoServer.serverPath + '/echo',
```

Comment:
@denis-kralj-novu I'm about to merge this PR: https://github.com/novuhq/novu/pull/5619/files and it was decided to use just `url` there. I believe we should go with it?